src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -12,6 +12,7 @@
                 <div>
                     <img id="{{'comment_avatar_' + counter}}"
                       class="user-assign-avatar pull-left" 
+                      title="{{comment.relationalData.creator?.attributes?.fullName}}"
```

Comment:
We are not using the default tooltip provided by HTML. 
We are using the ng2-bootstrap tooltip.
Please refer to this PR https://github.com/almighty/almighty-ui/pull/569.